src/library/objects/ShowcaseExporter.js

```diff
@@ -0,0 +1,881 @@
+                }
+
+                var sortingParam = this._paramToApi[KC3StrategyTabs.gears.definition._defaultCompareMethod[typeId]];
+                if (!!sortingParam) {
+                    masterIds.sort(compareEquip);
+                }
+
+                for (var i = 0; i < masterIds.length; i++) {
+                    masterId = masterIds[i];
+                    var canvasData = this._equipCanvases[masterId];
+                    ctx.drawImage(canvasData, 0, y, canvasData.width, canvasData.height);
+                    y += canvasData.height;
+
+                }
+                y += rowHeight * 0.5;
+
+                ctx.fillStyle = "#FFF";
+                for (var xPos = 0; xPos < 0 + canvas.width; xPos += 20) {
+                    ctx.fillRect(xPos, y - 2, 10, 2);
+                }
+            }
+        }
+    };
+
+    ShowcaseExporter.prototype._drawEquipGroup = function (group, rowWidth, rowHeight) {
+        var typeCount = 0;
+        for (var typeId in group.types) {
+            typeCount++;
+        }
+
+        var height = 0;
+        for (var i in group.types) {
+            if (i.startsWith("t")) {
+                height += this._drawEquipTypes(group.types[i], rowWidth, rowHeight);
+            }
+        }
+        if (typeCount > 1)
+            height += rowHeight * typeCount * 2;// + typename each
+        return height + rowHeight;// + groupName
+    };
+
+    ShowcaseExporter.prototype._drawEquipTypes = function (types, rowWidth, rowHeight) {
+        var height = 0;
+
+        for (var masterId in types.gears) {
+            if (!types.hasOwnProperty(masterId) && masterId.startsWith("m")) {
+                height += this._drawEquip(types.gears[masterId], rowWidth, rowHeight, false);
+            }
+        }
+        return height + 0.5 * rowHeight;
+    };
+
+    ShowcaseExporter.prototype._drawEquip = function (equip, rowWidth, rowHeight, fake) {
+        var height;
+        if (!fake) {
+            height = this._drawEquip(equip, rowWidth, rowHeight, true);
+        }
+
+        var img = this._equipTypeImages[KC3Master.slotitem(equip.masterId).api_type[3]];
+        var canvas = document.createElement("CANVAS"), ctx = canvas.getContext("2d");
+        var x = 0, y = 0;
+
+        if (fake === false) {
+            canvas.width = rowWidth;
+            canvas.height = height;
+            ctx.drawImage(img, 0, 0, img.width, img.height, x, y, rowHeight, rowHeight);
+        }
+
+        var equipMaxY = this._drawEquipInfo(equip, ctx, x, y, rowWidth, rowHeight, fake);
+
+        ctx.fillStyle = "#000";
+        var fontSize;
+        for (var i = 0; i <= 10; i++) {
+            if (i === 0)
+                fontSize = 14;
+            else
+                fontSize = 10;
+            if (equip["s" + i] === 0)
+                continue;
+
+            var text = "";
+            if (i === 0) {
+                text = " x";
+            } else if (i === 10) {
+                text = "+10★ x";
```

Comment:
ingame it shown as ★+10, maybe. btw this block seems no different with the next...